### DISCUSSION TITLE
Project configuration option 'justmycode "Off"' doesn't work

### DISCUSSION THREAD

**Discussion by bogez57** [premake/premake-core#1825] [Q&A]
In order to try and get things working properly with remedybg for a project I have to remove the generated msvc compiler flag "/jmc" from the output. I found on the premake site this flag is turned off with 'justmycode "Off"'. I tried putting this in multiple places within the premake file but keep getting this error:

Error: premake5.lua:69: attempt to call a nil value (global 'justmycode')

I'm not really familiar with premake and the website didn't really help with this https://premake.github.io/docs/justmycode/#examples so I'm wondering what I'm doing wrong.

Using premake 5.0.0-alpha14-windows

**Comment by nickclark2016** [comment]
Please try updating your version of Premake. The `justmycode` configuration was added in a later version.

**Reply by bogez57** [reply]
Okay, makes sense. Thanks
